Rakefile

```diff
@@ -192,9 +192,12 @@ end
 
 def providers
   if ENV['PROVIDERS'] == 'all'
-    ALLOWED_PROVIDERS
-  elsif ! ENV['PROVIDERS'].nil?
-    ENV['PROVIDERS'].split(',')
+    return ALLOWED_PROVIDERS
   end
-  ALLOWED_PROVIDERS
+
+  if not ENV['PROVIDERS'].nil?
+    return ENV['PROVIDERS']
```

Comment:
I was wondering whether we ever want to select more than one provider at a time right now. If we're only using 2 then "all" covers that case right?